hasty/client.py

```diff
@@ -40,13 +40,16 @@ def get_projects(self):
         """
         return PaginatedList(Project, self._requester, Project.endpoint)
 
-    def get_project(self, project_id):
+    def get_project(self, project_id: str, content_type: str = ProjectType.Image) -> Union[Project, VideoProject]:
         """
         Returns project :py:class:`~hasty.Project` by id
 
         Arguments:
             project_id (str): Project id
         """
+        if content_type == ProjectType.Video:
```

Comment:
As we discussed, let's infer content type from project data and attach the right class after.